robosuite/environments/door.py

```diff
@@ -304,7 +308,7 @@ def _reset_internal(self):
 
             # # Loop through all objects and reset their positions
             # for i, (obj_name, _) in enumerate(self.mujoco_objects.items()):
-            #     self.sim.data.set_joint_qpos(obj_name, np.concatenate([np.array(obj_pos[i]), np.array(obj_quat[i])]))
+            #     self.sim.data.set_joint_qpos(obj_name + "_jnt0", np.concatenate([np.array(obj_pos[i]), np.array(obj_quat[i])]))
```

Comment:
this looks like an automated name generation type thing. Was this intentional or was this just to make the sim work? Also, what is this referencing exactly? From above it seems like the Door object has no joints

Reply:
Yes you're right - I just made this modification in case you need to do any resetting for the door here - but we will be abandoning this reset scheme anyways. As for the name - it is automatic name generation for each generated joint per object - take a look at `TableTopTask` - it uses an objects ` joints` property to create desired joints per object, and when creating an object, you can pass a list of `joints` to create the joints for that object. By default a free joint is created per object, but for the door, I explicitly pass an empty list to ensure that no joint is created for the entire door object (the door xml creates hinge joints for the door and the handle).